ui/stops/OBADepartureTimeLabel.h

```diff
@@ -0,0 +1,16 @@
+//
+//  OBADepartureTimeLabel.h
+//  org.onebusaway.iphone
+//
+//  Created by Chad Royal on 9/25/16.
+//  Copyright © 2016 OneBusAway. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "OBADepartureStatus.h"
+
+@interface OBADepartureTimeLabel : UILabel
+
+-(void)renderTimeLabel:(NSString *)minutesUntilDeparture forStatus:(OBADepartureStatus)status;
```

Comment:
I feel like the closest analog to this in Cocoa Touch is the UIButton method `-setText:forState:`, so perhaps rename this to `-setText:forStatus:`?